fileversion.go

```diff
@@ -169,6 +169,28 @@ type FixedFileInfo struct {
 	FileSubtype    VerFileSubtype
 }
 
+type StringFileInfoNames string
```

Comment:
Let's rename this to `FileVersionString` so that it's clear that it relates to the version information.